mobile/fula_test.go

```diff
@@ -2,57 +2,76 @@ package fulaMobile_test
 
 import (
 	"context"
+	"math/rand"
 	"os"
 	"testing"
 
 	"github.com/functionland/go-fula/drive"
-	"github.com/functionland/go-fula/mobile"
+	fulaMobile "github.com/functionland/go-fula/mobile"
 	fileP "github.com/functionland/go-fula/protocols/file"
-	"github.com/ipfs/kubo/core"
-	"github.com/ipfs/kubo/core/bootstrap"
+
+	// "github.com/ipfs/kubo/core"
+	"github.com/libp2p/go-libp2p"
+	"github.com/libp2p/go-libp2p-core/crypto"
+	"github.com/libp2p/go-libp2p-core/host"
 	"github.com/libp2p/go-libp2p-core/network"
 	"github.com/libp2p/go-libp2p-core/peer"
-	routedhost "github.com/libp2p/go-libp2p/p2p/host/routed"
+	"github.com/libp2p/go-libp2p-core/peerstore"
 )
 
-func initNodes() (*core.IpfsNode, *fulaMobile.Fula, error) {
-	apis, nodes, err := fulaMobile.MakeAPISwarm(context.Background(), true, 2)
+func initNodes() (host.Host, *fulaMobile.Fula, error) {
+	apis, _, err := fulaMobile.MakeAPISwarm(context.Background(), true, 2)
 	if err != nil {
 		return nil, nil, err
 	}
 	fapi := apis[0]
-	node1 := nodes[0]
-	node2 := nodes[1]
+	// node1 := nodes[0]
+	// node2 := nodes[1]
```

Comment:
Please take a look at all commented out code here and either remove them or uncomment them. We can always get it back from version control system if removed.